akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -487,11 +487,46 @@ class Source[+Out, +Mat](delegate: scaladsl.Source[Out, Mat]) extends Graph[Sour
    * true, false,       // elements go into second substream
    * true, false, false // elements go into third substream
    * }}}
+   *
+   * If the split predicate `p` throws an exception and the supervision decision
+   * is [[akka.stream.Supervision.Stop]] the stream and substreams will be completed
+   * with failure.
+   *
+   * If the split predicate `p` throws an exception and the supervision decision
+   * is [[akka.stream.Supervision.Resume]] or [[akka.stream.Supervision.Restart]]
+   * the element is dropped and the stream and substreams continue.
+   *
+   * See also [[Source.splitAfter]].
```

Comment:
javadsl.Source had a bit of missing scaladoc, fixed. The rest is all consistent.